src/App.jsx

```diff
@@ -1,243 +1,22 @@
-          </div>
-       
-          <p className='heading_prof heading_org'>
-          The Jewish Agency for Israel, Jerusalem, Israel  ||  2005-2009
-          </p>
-          <div className="exp-text">
-            <p>Coordinator and Program Manager</p>
-          
-            As a program manager, I was responsible for the organization and coordination of short-
-            term programs of the Jewish Agency in Israel. That included mostly programs for
-            volunteering in IDF bases and professional seminars for non-Israeli citizens. At the
-            same time, I coordinated and organized "Information days" and "Information forums" for
-            different kinds of the target audience.
-            An organizational work included logistic orders, content preparation, partnership with
-            tour guides and group counselors, and also cooperation with IDF, Israeli authorities, and
-            numerous Jewish Agency branches in Israel and abroad.
-            I was also in charge of documentation preparation based on different programs and
-            systems including ERP, CMS, MS and Office programs.
-          </div>
 
-          <div className='experience'> 
-          <p className='heading heading_prof'>Professional Courses and skills </p></div>
-          <hr className='line'></hr>
-          <p className='heading_prof heading_org'>Courses and certificates</p>
-            <div className='exp-text'>
-              <img className='arrow_icon' src={right_arrow} alt="arrow" />
-              <span >React- The Complete Guide</span></div>
-            <div className='exp-text'> 
-            <img className='arrow_icon' src={right_arrow} alt="arrow" />
-            <span>The Complete 2023 Web Development Bootcamp</span></div>
-            <div className=' exp-text'>
-            <img className='arrow_icon' src={right_arrow} alt="arrow" /> <span>The Complete JavaScript Course 2022: From Zero to Expert!</span>
-            </div>
-            <div className=' exp-text'>
-            <img className='arrow_icon' src={right_arrow} alt="arrow" /> 
-            <span>Accelerated ES6 JavaScript Training</span>
-            </div>
-            <div className=' exp-text'>
-            <img className='arrow_icon' src={right_arrow} alt="arrow" />
-            <span>CSS - The Complete Guide 2022 (incl. Flexbox, Grid & Sass)</span></div>
-            <div className=' exp-text'>
-            <img className='arrow_icon' src={right_arrow} alt="arrow" />
-            <span>Advanced CSS and Sass: Flexbox, Grid, Animations and More!</span></div>
-            <div className=' exp-text'>
-            <img className='arrow_icon' src={right_arrow} alt="arrow" />
-            <span>HTML & CSS Course for the beginners</span></div>
+  return (
+    <div className='container'>
+        <div>
+          <Routes>
+            <Route path="/en" Component={Resume}></Route>
+            <Route path="/de" Component={Resume}></Route>
```

Comment:
Das Problem ist gerade so, dass man überhaupt /de schreirben muss. Ansonsten kriegt er eine leere Seite.
Du kannst das mit dem `<Navigate replace to="/de" />` machen, damit wenn keines Route gefunden wurde, das Komponent wird Re-route zum '/de' machen.

Reply:
Habe gemacht, mit diesem Rout:
<Route path="/" element={<Navigate to="/de"replace/>}></Route>